go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -144,10 +144,10 @@ func createNode(name, os, ip string, annotations map[string]string) *v1.Node {
 }
 
 func addSyncFlows(fexec *ovntest.FakeExec) {
-	fexec.AddFakeCmdsNoOutputNoError([]string{
+	fexec.AddFakeCmdsNoOutputNoErrorMayDuplicate([]string{
 		"ovs-ofctl dump-flows --no-stats br-ext table=20",
 		"ovs-ofctl -O OpenFlow13 --bundle replace-flows br-ext -",
-	})
+	}, true)
```

Comment:
I have a somewhat bad feeling about allowing duplicates of write commands and not just only read commands.

Reply:
> I have a somewhat bad feeling about allowing duplicates of write commands and not just only read commands.

Yes, that is a super valid point here. I found the reason why this is happening, but did a really bad job in describing it here. I will break this PR into 2 commits to separate the minor enhancement from the change in the test. Then, on the second one, I will better articulate the reason why the test change is needed. To be continued. ;)
TY @jcaamano and @tssurya !

Reply:
Added info about allowing duplicates here: 
https://github.com/ovn-org/ovn-kubernetes/pull/3610#issuecomment-1561756856